tools/workspace-plugin/src/generators/normalize-package-dependencies/index.spec.ts

```diff
@@ -43,6 +44,9 @@ describe('normalize-package-dependencies generator', () => {
   const infoLogSpy = jest.spyOn(console, 'info').mockImplementation(noop);
 
   beforeEach(() => {
+    graphMock = {
```

Comment:
making graphMock always empty object before test exec, previously this var was mutated between tests